paper/draft.md

```diff
@@ -0,0 +1,28 @@
+
+### Statement of need
+The JSON data model is a popular way to represent structured data. It is attractive because of its relative simplicity and is a natural fit with the standard types of many programming languages. However, this simplicity comes at the cost that basic JSON lacks expressive features useful for working with complex data structures and document formats, such as schemas, object references, and namespaces.
+
+JSON-LD is a W3C standard providing a way to describe how to interpret a JSON document as Linked Data by means of a "context". JSON-LD provides a powerful solution for representing object references and namespaces in JSON based on standard web URIs but is not itself a schema language. Without a schema providing a well-defined structure, it is difficult to process an arbitrary JSON-LD document as idiomatic JSON because there are many ways to express the same data that are logically equivalent but structurally distinct.
+
+### State of the field
+>It's better to extend or rework this section as _references should include full names of venues, e.g., journals and conferences, not abbreviations only understood in the context of a specific discipline_
+
+Several schema languages exist for describing and validating JSON data, such as JSON Schema and Apache Avro data serialization system, however, none understand linked data. As a result, to fully take advantage of JSON-LD to build the next generation of linked data applications, one must maintain separate JSON schema, JSON-LD context, RDF schema, and human documentation, despite the significant overlap of content and obvious need for these documents to stay synchronized.
+
+Schema Salad is designed to address this gap. It provides a schema language and processing rules for describing structured JSON content permitting URI resolution and strict document validation. The schema language supports linked data through annotations that describe the linked data interpretation of the content, enables generation of JSON-LD context and RDF schema, and production of RDF triples by applying the JSON-LD context. The schema language also provides for robust support of inline documentation.
+
+### Mentions
+> Put here recently submitter CWL paper. The title of this section should be changed into something meaningful.
+
+### Examples
+> I think it's would be great to put here some of the schema-salad examples (if we are still within 1000 words limit)
```

Comment:
@pjotrp can you say a few works about the use of schema-salad in PubSeq?

Reply:
Yes. Let me have a look.

Reply:
I think the reader would be best served with a few examples. Show a JSON record and a mapping and the RDF or JSON-LD output. I mean the strength of schema salad is a simple translation with error checking. To be honest, I only understood what schema salad was about when I ran something. That is how you can help the interested user. Does that make sense? We can use examples from PubSeq for sure. Does that make sense? If you inject the images as figures they don't count as words.

Reply:
It would be nice to have PubSeq examples in this paper. We can also mention PubSeq in the **Mentions** section (this section will be renamed later, but it's for a _representative set of past or ongoing research projects using the software and recent scholarly publications enabled by it_). As for the figures, they shouldn't be counted as words. Here is an example of how they add figures into the papers

# Figures

Figures can be included like this:
![Caption for example figure.\label{fig:example}](figure.png)
and referenced from text using \autoref{fig:example}.

Figure sizes can be customized by adding an optional second parameter:
![Caption for example figure.](figure.png){ width=20% }

Reply:
I am proposing to turn examples (YAML/JSON/RDF) into figures, so they don't count as words.